converters/fromZigbee.js

```diff
@@ -8308,6 +8308,18 @@ const converters = {
             return result;
         },
     },
+    WS_USC02_single_and_double_press: {
```

Comment:
Can this be merged with `xiaomi_multistate_action`?

Reply:
Done